README.md

```diff
@@ -247,94 +251,94 @@ See Strava API docs for returned data structures.
+- `strava.athlete.listRoutes(args,done)`
+- `strava.athlete.listClubs(args,done)`
+- `strava.athlete.listZones(args,done)`
 
 #### Athletes
 
-* `strava.athletes.get(args,done)` *Get a single activity. args.id is required*
-* `strava.athletes.stats(args,done)`
+- `strava.athletes.get(args,done)` _Get a single activity. args.id is required_
+- `strava.athletes.stats(args,done)`
 
 #### Activities
 
-* `strava.activities.get(args,done)`
-* `strava.activities.create(args,done)`
-* `strava.activities.update(args,done)`
-* `strava.activities.listFriends(args,done)`
-* `strava.activities.listZones(args,done)`
-* `strava.activities.listLaps(args,done)`
-* `strava.activities.listComments(args,done)`
-* `strava.activities.listKudos(args,done)`
-* `strava.activities.listPhotos(args,done)`
-* `strava.activities.listRelated(args,done)`
+- `strava.activities.get(args,done)`
+- `strava.activities.create(args,done)`
+- `strava.activities.update(args,done)`
+- `strava.activities.listFriends(args,done)`
+- `strava.activities.listZones(args,done)`
+- `strava.activities.listLaps(args,done)`
+- `strava.activities.listComments(args,done)`
+- `strava.activities.listKudos(args,done)`
+- `strava.activities.listPhotos(args,done)`
+- `strava.activities.listRelated(args,done)`
 
 #### Clubs
 
-* `strava.clubs.get(args,done)`
-* `strava.clubs.listMembers(args,done)`
-* `strava.clubs.listActivities(args,done)`
-* `strava.clubs.listAnnouncements(args,done)`
-* `strava.clubs.listEvents(args,done)`
-* `strava.clubs.listAdmins(args,done)`
-* `strava.clubs.joinClub(args,done)`
-* `strava.clubs.leaveClub(args,done)`
+- `strava.clubs.get(args,done)`
+- `strava.clubs.listMembers(args,done)`
+- `strava.clubs.listActivities(args,done)`
+- `strava.clubs.listAnnouncements(args,done)`
+- `strava.clubs.listEvents(args,done)`
+- `strava.clubs.listAdmins(args,done)`
+- `strava.clubs.joinClub(args,done)`
+- `strava.clubs.leaveClub(args,done)`
 
 #### Gear
 
-* `strava.gear.get(args,done)`
+- `strava.gear.get(args,done)`
 
 #### Push Subscriptions
 
 These methods Authenticate with a Client ID and Client Secret. Since they don't
 use OAuth, they are not available on the `client` object.
 
- * `strava.pushSubscriptions.list({},done)`
- * `strava.pushSubscriptions.create({callback_url:...},done)`
- *  We set 'object\_type to "activity" and "aspect\_type" to "create" for you.
- * `strava.pushSubscriptions.delete({id:...},done)`
+- `strava.pushSubscriptions.list({},done)`
+- `strava.pushSubscriptions.create({callback_url:...},done)`
+- We set 'object_type to "activity" and "aspect_type" to "create" for you.
+- `strava.pushSubscriptions.delete({id:...},done)`
 
 #### Running Races
 
- * `strava.runningRaces.get(args,done)`
- * `strava.runningRaces.listRaces(args,done)`
+- `strava.runningRaces.get(args,done)`
+- `strava.runningRaces.listRaces(args,done)`
 
 #### Routes
 
-
- * `strava.routes.get(args,done)`
+- `strava.routes.get(args,done)`
+- `strava.routes.getFile(args, done)`
```

Comment:
Although the documentation for existing methods is often poor, let's do better with new routes, by showing here that the args for `getFile()` are `id` and `file_type`.

Reply:
Also, please be consistent abut whether there is a space ",done" or ", done". Looks like the rest of the file doesn't use a space.